openvdb/openvdb/tools/MeshToVolume.h

```diff
@@ -1132,7 +1133,7 @@ inline void
 fillArray(ValueType* array, const ValueType val, const size_t length)
 {
     const auto grainSize = std::max<size_t>(
-        length / tbb::task_scheduler_init::default_num_threads(), 1024);
+        length / tbb::this_task_arena::max_concurrency(), 1024);
```

Comment:
**Significant change**

This should now be affected by the caller configuring their own arena